packages/create-app/template-ef/src/main.css

```
@@ -0,0 +1,44 @@
+@import url('https://fonts.googleapis.com/css?family=Blinker&display=swap');
+
+html {
+	font-family: 'Blinker', sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
```

Something wrong with indentation here.

```suggestion
html {
	font-family: 'Blinker', sans-serif;
	-webkit-font-smoothing: antialiased;
	-moz-osx-font-smoothing: grayscale;
}
```